doc/src/getting-started-guide.md

```diff
@@ -137,203 +157,266 @@ When the AtomVM virtual machine starts, it will search for the first module that
 
-### Running Hello Pico
+These instructions cover how to provision the AtomVM virtual machine onto your development machine.  Running applications locally can sometimes be a useful exercise in debugging.
 
-This example will print a Hello Pico message repeatedly.
+> Note.  Not all programming interfaces are supported on all platforms.  See the AtomVM [Programmers Guide](programmers-guide.md) for more information.
 
-It is built into `build/examples/erlang/rp2040/hello_pico.uf2`.
+For most applications, you should only need to install the VM once (or at least once per desired AtomVM release).  Once the VM is installed, you can then begin development of Erlang or Elixir applications, which can then be flashed as part of your routine development cycle.
 
-You can install it and then connect to the serial port with minicom.
+### Requirements
 
-### Running your own BEAM code on Raspberry Pico
+Deployment of AtomVM on the Generic UNIX platform requires the following components:
 
-You need to create an avm file using the `packbeam` tool the [`atomvm_rebar3_plugin`](https://github.com/atomvm/atomvm_rebar3_plugin).
+* A computer running MacOS or Linux (Windows support is not currently supported);
+* An [Erlang/OTP](https://erlang.org) and compatible [Elixir](https://elixir-lang.org) runtime;
+* (recommended) For Erlang programs, [`rebar3`](https://rebar3.org);
+* (recommended) For Elixir programs, [`mix`](https://elixir-lang.org/getting-started/mix-otp/introduction-to-mix.html), which ships with the Elixir runtime;
 
-    packbeam create -p -i packed.avm module.beam
+For information about specific versions of required software, see the AtomVM [Release Notes](./release-notes.md).
 
-or
+### Installing the AtomVM virtual machine
 
-    rebar3 packbeam -p -i packed.avm module.beam
+The following methods can be used to install the AtomVM virtual machine on the Generic UNIX platform:
 
-Then the BEAM file must be converted to UF2.
-The VM currently expects the application to be loaded at address 0x10100000.
+1. Download Linux Binaries
+1. (MacOS only) Installing via [macports](https://www.macports.org) or [Homebrew](https://brew.sh);
+1. Building from source.
 
-    ./uf2tool create -o packed.uf2 -s 0x10100000 packed.avm
+#### Installation on Linux Platforms
 
-Copy this UF2 to the Pico after you copied the VM (`AtomVM.uf2`) and the
-standard libraries (`atomvmlib.uf2`).
+Downloading a pre-built binary image for Linux is by far the easiest path to getting started with development on a Linux development machine.  Binary images contain the virtual machine.
 
+Download a stable or latest development [release image](https://github.com/atomvm/AtomVM/releases) for Linux.
 
-## Getting Started on the Generic UNIX platform
+This image will generally take the form:
+
+    Atomvm-linux-<arch>-<atomvm-version>.img
```

Comment:
There is currently no file extension on the binary executable.

Reply:
fixed